core/templates/dev/head/components/RatingDisplayDirective.js

```diff
@@ -60,9 +60,9 @@ oppia.directive('ratingDisplay', [
           for (var i = 0; i < $scope.stars.length; i++) {
             $scope.stars[i].cssClass = (
               ratingValue === undefined ? 'fa-star-o' :
-              ratingValue < $scope.stars[i].value - 0.75 ? 'fa-star-o' :
-              ratingValue < $scope.stars[i].value - 0.25 ? 'fa-star-half-o' :
-              'fa-star');
+                ratingValue < $scope.stars[i].value - 0.75 ? 'fa-star-o' :
```

Comment:
Is there a way to force indent to happen only after "(", "[" and "{", but not be triggered by ":" or "?" or "&&", or ...?

Reply:
I did look into this. Eslint does not provide any such option to force indent.  
Thanks!

Reply:
Sorry -- what I meant was more along the lines of, preventing additional indenting after ":", "?", etc. I'm not sure it makes sense for those things to start a "new scope", as brackets do.

Did this behaviour change recently with eslint? Is it worth writing a custom rule?

Reply:
I found out that there exists an option for ternary expressions, `flatTernaryExpresions` which is false by default. I have set it to true now. This would fix this problem.